WooCommerce/Classes/ViewModels/CardPresentPayments/PaymentCaptureOrchestrator.swift

```diff
@@ -36,75 +36,59 @@ final class PaymentCaptureOrchestrator {
+        /// reader begins to collect payment.
+        ///
+        suppressPassPresentation()
 
-            /// Briefly suppress pass (wallet) presentation so that the merchant doesn't attempt to pay for the buyer's order when the
-            /// reader begins to collect payment.
-            ///
-            suppressPassPresentation()
-
-            let paymentAction = CardPresentPaymentAction.collectPayment(
-                siteID: order.siteID,
-                orderID: order.orderID,
-                parameters: parameters,
-                onCardReaderMessage: { (event) in
-                    switch event {
+        let paymentAction = CardPresentPaymentAction.collectPayment(
+            siteID: order.siteID,
+            orderID: order.orderID,
+            parameters: parameters,
+            onCardReaderMessage: {(event) in
+                switch event {
                     case .waitingForInput:
                         onWaitingForInput()
                     case .displayMessage(let message):
                         onDisplayMessage(message)
                     default:
                         break
-                    }
-                },
-                onCompletion: { [weak self] result in
-                    self?.allowPassPresentation()
-                    onProcessingMessage()
-                    self?.completePaymentIntentCapture(
-                        order: order,
-                        captureResult: result,
-                        onCompletion: onCompletion
-                    )
                 }
-            )
-
-            ServiceLocator.stores.dispatch(paymentAction)
-        }
+            },
+            onCompletion: {[weak self] result in
+                self?.allowPassPresentation()
+                onProcessingMessage()
+                self?.completePaymentIntentCapture(
+                    order: order,
+                    captureResult: result,
+                    onCompletion: onCompletion
+                )
+            }
+        )
 
-        ServiceLocator.stores.dispatch(customerAction)
+        ServiceLocator.stores.dispatch(paymentAction)
     }
 
     func cancelPayment(onCompletion: @escaping (Result<Void, Error>) -> Void) {
-        let action = CardPresentPaymentAction.cancelPayment() { [weak self] result in
+        let action = CardPresentPaymentAction.cancelPayment() {[weak self] result in
             self?.allowPassPresentation()
             onCompletion(result)
         }
         ServiceLocator.stores.dispatch(action)
     }
 
     func emailReceipt(for order: Order, params: CardPresentReceiptParameters, onContent: @escaping (String) -> Void) {
-        let action = ReceiptAction.generateContent(order: order, parameters: params) { emailContent in
+        let action = ReceiptAction.generateContent(order: order, parameters: params) {emailContent in
```

Comment:
super nit: not sure if this was from some Xcode settings if you're using Xcode?
```suggestion
        let action = ReceiptAction.generateContent(order: order, parameters: params) { emailContent in
```